packages/babel-plugin-named-asset-import/.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+  "presets": ["@babel/preset-env"]
```

Comment:
Is this required? I don't see anything like this in the `babel-plugin-tester` docs.

Reply:
It is required to transplie tests which are implemented in es6. I will update tests and remove all not needed babel dependencies